pandas/io/excel/_odfreader.py

```diff
@@ -1,5 +1,7 @@
 from typing import List
 
+import numpy as np
```

Comment:
Is there a reason for the changes to the reader here? If solving a separate issue should do in a separate PR

Reply:
This came up when doing roundtrip tests. I had to fix this one or else every 0 in the sheet would be removed.

Reply:
Can you just add a quick what’s new for this too?

Reply:
Added